tests/qos/test_pfc_pause.py

```diff
@@ -0,0 +1,396 @@
+
+from ansible_host import AnsibleHost
+from qos_fixtures import *
+from qos_helpers import ansible_stdout_to_str, eos_to_linux_intf, get_active_intfs, get_addrs_in_subnet, get_neigh_ip,check_mac_table, get_mac, config_intf_ip_mac
+import pytest
+import os
+import time
+import re
+import struct
+import random
+import ipaddress
+
+PFC_GEN_FILE = 'pfc_gen.py'
+PFC_GEN_FILE_RELATIVE_PATH = '../../ansible/roles/test/files/helpers/pfc_gen.py'
+PFC_GEN_FILE_DEST = '~/pfc_gen.py'
+PFC_PKT_COUNT = 1000000000
+
+PTF_FILE_RELATIVE_PATH = '../../ansible/roles/test/files/ptftests/pfc_pause_test.py'
+PTF_FILE_DEST = '~/pfc_pause_test.py'
+PTF_PKT_COUNT = 50
+PTF_PKT_INTVL_SEC = 0.1
+PTF_PASS_RATIO_THRESH = 0.75
+
+""" Maximum number of interfaces to test on a DUT """
+MAX_TEST_INTFS_COUNT = 4
+         
+def atoi(text):
+    return int(text) if text.isdigit() else text
+
+def natural_keys(text):
+    return [atoi(c) for c in re.split(r'(\d+)', text)]
+
+def setup_testbed(ansible_adhoc, testbed, leaf_fanouts):
```

Comment:
can you move this function to the helper file? These steps will be applicable to multiple PFC tests

Reply:
@neethajohn Fixed.